internal/command/orgs/orgs.go

```diff
@@ -78,23 +78,23 @@ func slugFromFirstArgOrSelect(ctx context.Context) (slug string, err error) {
 	client := client.FromContext(ctx).API()
 
 	var orgs []api.Organization
-	if orgs, err = client.GetOrganizations(ctx); err != nil {
+	if orgs, err = client.GetOrganizations(ctx, filters...); err != nil {
 		return
 	}
 	sort.OrganizationsByTypeAndName(orgs)
 
 	var org *api.Organization
 	if org, err = prompt.SelectOrg(ctx, orgs); prompt.IsNonInteractive(err) {
 		err = errSlugArgMustBeSpecified
-	} else {
+	} else if err == nil {
```

Comment:
was it crashing on errors before?

Reply:
It panicked if you ^C'ed at the prompt, yeah.

Reply:
nice